src/Features/Core/Portable/NavigateTo/AbstractNavigateToSearchService.cs

```diff
@@ -40,123 +37,5 @@ internal abstract partial class AbstractNavigateToSearchService : INavigateToSea
                     await onResultFound(result).ConfigureAwait(false);
             };
         }
-
-        public async Task SearchDocumentAsync(Document document, string searchPattern, IImmutableSet<string> kinds, bool isFullyLoaded, Func<INavigateToSearchResult, Task> onResultFound, CancellationToken cancellationToken)
-        {
-            if (isFullyLoaded)
-            {
-                await SearchFullyLoadedDocumentAsync(document, searchPattern, kinds, onResultFound, cancellationToken).ConfigureAwait(false);
-            }
-            else
-            {
-                await SearchCachedDocumentsAsync(ImmutableArray.Create(document), ImmutableArray<Document>.Empty, searchPattern, kinds, onResultFound, cancellationToken).ConfigureAwait(false);
-            }
```

Comment:
we used to have one codepath that woudl operate on a project, passing a bool to control which fo these paths to go down.  Now, this is just exposed through separate SearchProjectAsync and SearchCachedDocumentsASync entrypoints.